### PR TITLE
[ioctl] build node command into new ioctl

### LINKED ISSUE #3428: [ioctl] build node command into new ioctl
### What would you like to be added
 - Step 1: create `node.go` and `node_test.go` files in `./ioctl/newcmd/node`. create this directory if not exist. 
 - Step 2: refactor  command  https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/node/node.go in `node.go`
 - Step 3: add unit test in node_test.go

#### Notes:
1. Use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to construct the Cobra command.
2. [Output package](https://github.com/iotexproject/iotex-core/blob/master/ioctl/output) is deprecated. `e.g.` `output.PrintError` could be replaced with `errors.Wrap`.
3. Global variables (config.ReadConfig  config.DefaultConfigFile etc.) have been wrapped into client interface. Please use [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) to access them if needed
4. The logic of new code should be similar to [the old one](https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/action/xrc20transferfrom.go).
5. In each unit test, the results should be checked whether they are as the expected. 

#### Example:
1. https://github.com/iotexproject/iotex-core/blob/master/ioctl/newcmd/account/account.go

### Why is this needed
Unit tests couldn't be added for the old ioctl command (./ioctl/cmd)

### PR DESCRIPTION
**Pull request by pocockn**
# Description

There is no command within the new node package.

Fixes #3428 

## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] make test

@huof6829  is this more along the lines of what you were thinking?

### CODE REVIEW

File: ioctl/newcmd/node/node.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package node
+
+import (
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/spf13/cobra"
```
Comment (huof6829) [inline ioctl/newcmd/node/node.go:6]: line6 should be a single group

File: ioctl/newcmd/node/node_test.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package node
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
```
Comment (huof6829) [inline ioctl/newcmd/node/node_test.go:7]: `iotex-core`  should be  a single group

**Comment by huof6829** [comment]
@pocockn   Yes. it's right.  Thanks for your task!

File: ioctl/newcmd/node/node.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		Use:   nodeUses,
+		Short: nodeShorts,
+	}
+
+	nc.AddCommand(NewNodeDelegateCmd(client))
+	nc.AddCommand(NewNodeRewardCmd(client))
+
+	client.SetEndpointWithFlag(nc.PersistentFlags().StringVar)
+	client.SetInsecureWithFlag(nc.PersistentFlags().BoolVar)
+
```
Comment (huof6829) [inline ioctl/newcmd/node/node.go:36]: add flag.
https://github.com/iotexproject/iotex-core/blob/ba24a74d0432e4440880d60dcc09c6d8214e4e65/ioctl/cmd/node/node.go#L51

**Comment by pocockn** [comment]
> @pocockn Yes. it's right. Thanks for your task!

Thanks! Imports fixed.

**Review by Liuhaai — APPROVED** [review]

**Comment by Liuhaai** [comment]
thx for contribution

**Pull request merged**